### PR TITLE
feat: config health

### PR DESCRIPTION
**Pull request by vinhphuctadang**
We need these config for new health endpoint in background processes services

### CODE REVIEW

File: docker/addons/docker-compose.dex.yaml [hunk 1]
```diff
@@ -57,9 +57,19 @@ services:
       INDEXER_CHRONOS_LEADERBOARD_CROSS_MARKET_IDS: ${INDEXER_CHRONOS_LEADERBOARD_CROSS_MARKET_IDS}
       INDEXER_CHRONOS_LEADERBOARD_CUSTOM_RES: ${INDEXER_CHRONOS_LEADERBOARD_CUSTOM_RES}
       INDEXER_VOLUME_SINGLE_SIDE_HEIGHT: ${INDEXER_VOLUME_SINGLE_SIDE_HEIGHT}
+
```
Comment (mmeloni) [inline docker/addons/docker-compose.dex.yaml:60]: INDEXER_CHRONOS_RPC_ADDR: tcp://0.0.0.0:5500 :-> INDEXER_CHRONOS_RPC_ADDR: 0.0.0.0:5500

**Review by mmeloni — CHANGES_REQUESTED** [review]

File: docker/addons/docker-compose.dex.yaml [hunk 2]
```diff
@@ -57,9 +57,19 @@ services:
...
+      INDEXER_HTTP_LISTEN_ADDR: 0.0.0.0:9910
+      INDEXER_GRPC_LISTEN_ADDR: 0.0.0.0:4444
+      INDEXER_HTTP_TLS_CERT: /apps/data/public.crt
+      INDEXER_HTTP_TLS_KEY: /apps/data/private.key
+      INDEXER_HTTP_HORACLE_ADDR: ${INDEXER_HTTP_HORACLE_ADDR}
```
Comment (mmeloni) [inline docker/addons/docker-compose.dex.yaml:65]: here we can uniform with other vars:
INDEXER_HTTP_HORACLE_ADDR: https://k8s.${NETWORK}.horacle.injective.network/height/latest

Comment (vinhphuctadang) [reply docker/addons/docker-compose.dex.yaml:65]: We can input whole string, it's more flexible I think

**Review by mmeloni — APPROVED** [review]

File: docker/addons/docker-compose.dex.yaml [hunk 3]
```diff
@@ -281,8 +301,18 @@ services:
       INDEXER_BLOCK_FETCH_JOBS: ${INDEXER_EVENTPROVIDER_BLOCK_FETCH_JOBS}
       INDEXER_CHAIN_START_HEIGHT: ${INDEXER_CHAIN_START_HEIGHT}
       INDEXER_COSMOS_TLS_ENABLED: ${INDEXER_COSMOS_TLS_ENABLED}
+      
+      INDEXER_GRPC_LISTEN_ADDR: 0.0.0.0:9910
+      INDEXER_HTTP_LISTEN_ADDR: 0.0.0.0:4444
+      INDEXER_HTTP_TLS_CERT: /apps/data/public.crt
+      INDEXER_HTTP_TLS_KEY: /apps/data/private.key
+      INDEXER_HTTP_HORACLE_ADDR: https://k8s.${NETWORK}.horacle.injective.network/height/latest
```
Comment (Ri-go) [inline docker/addons/docker-compose.dex.yaml:309]: why not use `INDEXER_HTTP_HORACLE_ADDR` here?

**Pull request merged**